app/models/page_sequence.rb

```diff
@@ -14,7 +14,8 @@ class PageSequence
     "still-teaching",
     "current-school",
     "subjects-taught",
-    "mostly-teaching-eligible-subjects",
+    "leadership-position",
+    "mostly-not-teaching",
```

Comment:
The "not" in this slug I think makes it hard to parse. Is there a phrasing that works without a "not"?

Reply:
Changed to talk about the leadership question it asks.